tests/step_implementers/package/test_maven_package.py

```diff
@@ -1,339 +1,590 @@
 import os
+from pathlib import Path
 
-import pytest
 from testfixtures import TempDirectory
-import yaml
+
+import sh
```

Comment:
is sh needed?

Reply:
hmmm. i think not.

Reply:
removed.

Reply:
it is needed. cuz we reference `sh.ErrorReturnCode` for mocking that.